building/Windows.md

```diff
@@ -0,0 +1,133 @@
+# Windows
+
+If we want we can build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows with Cygwin and WSL.
+
+## Cygwin
+
+In the beginning, we need to download libUSB and set the QEMU environment.
+
+To do that we need to simply download QEMU from the official website which is targeted to our system. The same
+situation is with libUSB, but we need to follow the instructions provided by the creator.
+
+* [QEMU](https://www.qemu.org/download/#windows)
+* [libUSB](https://github.com/libusb/libusb)

```

Comment:
I would create a step-by-step instruction on how to install this tools. I know it may seem simple and unnecessary, but I would stick to the convention that it should be accessible to everyone. 

So I would use dropdown lists, for example, as it's done in building:
![image](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/116297248/72e730c2-3d75-4f1f-89d2-1026faab94aa)